iothub/device/src/Serialization/PayloadConvention.cs

```diff
@@ -1,35 +1,60 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+
 namespace Microsoft.Azure.Devices.Client
 {
     /// <summary>
-    /// The payload convention class. It is used to define a specific serializer as well as a specific content encoding.
+    /// The payload convention class.
     /// </summary>
     public abstract class PayloadConvention
```

Comment:
Imports all the relevant API surface from the PayloadSerializer and PayloadEncoder types.